tensorflow/lite/micro/tools/make/make_generate_cc_arrays.py

```diff
@@ -0,0 +1,55 @@
+# Lint as: python2, python3
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Generates cc and header files from bitmap or tflite files."""
+
+import argparse
+import os
+
+import generate_cc_arrays
+
+
+def main():
+    """Create cc sources with c arrays with data from each .tflite or .bmp."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument('output_dir', help='base directory for all outputs')
+    parser.add_argument('input_files',
+                        nargs='+',
+                        help='input bmp or tflite files to convert')
+    args = parser.parse_args()
+    for input_file in args.input_files:
```

Comment:
if the same file is listed multiple times in args.input_list it will be generated multiple times as well, right? So MICROLITE_CC_SRCS can have the same path repeated multiple times?

Reply:
Added dedup in this loop.